.github/workflows/CI.yml

```diff
@@ -52,6 +52,7 @@ jobs:
       PYVER: ${{ matrix.python-version }}
       OE_LICENSE: ${{ github.workspace }}/oe_license.txt
       PACKAGE: openforcefield
+      PYTEST_ARGS: -r fE --tb=short --cov=openforcefield --cov-config=setup.cfg --cov-append --cov-report=xml
```

Comment:
`setup.cfg` or another config file is probably a better home for these settings, but the docs I was digging through were hard to parse and this should work well enough